internal/services/v1/experimental.go

```diff
@@ -401,10 +427,6 @@ func (es *experimentalServer) BulkExportRelationships(
 				return es.rewriteError(ctx, err)
 			}
 		}
-
-		// Datastore namespace order might not be exactly the same as go namespace order
-		// so we shouldn't assume cursors are valid across namespaces
-		cur = nil
```

Comment:
can you clarify what happened here? I see a similar change was introduced earlier in the loop to make sure the cursor is reset for each iteration as the namespace changes

Reply:
Not sure I follow